book/src/SUMMARY.md

```diff
@@ -25,9 +25,4 @@ SPDX-License-Identifier: MIT OR Apache-2.0
     - [extern "C++Qt"](./bridge/extern_cppqt.md)
     - [Shared types](./bridge/shared_types.md)
     - [Attributes](./bridge/attributes.md)
-- [Reference: traits](./traits/index.md)
-    - [CxxQtType](./traits/cxxqttype.md)
-    - [Constructor](./traits/constructor.md)
-    - [Initialize](./traits/initialize.md)
-    - [Locking](./traits/locking.md)
-    - [Threading](./traits/threading.md)
```

Comment:
wonder if we should have a page in concept explaining threading and locking

Reply:
I think it's fine if the explanation lives on `docs.rs`.